documentation/modules/configuring/proc-config-mirrormaker2-securing-connection.adoc

```diff
@@ -162,6 +210,146 @@ spec:
+        operation: Create
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-status
+        operation: Read
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-status
+        operation: Describe
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-status
+        operation: DescribeConfigs
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-status
+        operation: Write
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-status
+        operation: Create
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-offsets
+        operation: Read
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-offsets
+        operation: Write
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-offsets
+        operation: Describe
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-offsets
+        operation: DescribeConfigs
+      - resource:
+          type: topic
+          name: mirrormaker2-cluster-offsets
+        operation: Create
+      # MirrorSourceConnector
+      - resource: # Needed for every topic which is mirrored
+          type: topic
+          name: "*"
+        operation: Create
+      - resource: # Needed for every topic which is mirrored
+          type: topic
+          name: "*"
+        operation: Alter
+      - resource: # Needed for every topic which is mirrored
+          type: topic
+          name: "*"
+        operation: AlterConfigs
+      - resource: # Needed for every topic which is mirrored
+          type: topic
+          name: "*"
+        operation: Write
+      # MirrorCheckpointConnector
+      - resource:
+          type: cluster
+        operation: Describe
+      - resource:
+          type: topic
+          name: my-source-cluster.checkpoints.internal
+        operation: Create
+      - resource:
+          type: topic
+          name: my-source-cluster.checkpoints.internal
+        operation: Describe
+      - resource:
+          type: topic
+          name: my-source-cluster.checkpoints.internal
+        operation: Write
+      - resource: # Needed for every group for which the offset is synced
+          type: group
+          name: "*"
+        operation: Read
+      - resource: # Needed for every group for which the offset is synced
+          type: group
+          name: "*"
+        operation: Describe
+      - resource: # Needed for every topic which is mirrored
+          type: topic
+          name: "*"
+        operation: Read
+      # MirrorHeartbeatConnector
+      - resource:
+          type: topic
+          name: heartbeats
+        operation: Create
+      - resource:
+          type: topic
+          name: heartbeats
+        operation: Describe
+      - resource:
+          type: topic
+          name: heartbeats
+        operation: Write
+----
++
```

Comment:
We need to make sure it is clear to the users that each user has to be create in different namespace.